src/components/Spacer.svelte

```diff
@@ -0,0 +1,8 @@
+<div class="shrink-0" />
```

Comment:
The spacer element is pretty funny. Are you sure that it is better than using flexbox?

Reply:
Used gap instead for flex items. Kept the Spacer part for the graph though as it'll be removed once the graph has been implemented.